crates/orchestrator/src/tests/workers/update_state/mod.rs

```diff
@@ -70,12 +70,12 @@ async fn test_update_state_worker(
         // creation)
         let completed_jobs =
             get_job_by_mock_id_vector(JobType::ProofCreation, JobStatus::Completed, number_of_processed_jobs as u64, 2);
-        for job in completed_jobs {
-            db.expect_get_job_by_internal_id_and_type()
-                .times(1)
-                .with(eq(job.internal_id.to_string()), eq(JobType::StateTransition))
-                .returning(|_, _| Ok(None));
-        }
+        // for job in completed_jobs {
```

Comment:
The test case was expecting many instances of get_job_by_internal_id_and_type because of the loop, which were not being used since only the first one was being used we corrected it to only expect for 1st element of the vector. Removed the comment.